### DISCUSSION TITLE
@vercel/node Environment Variable Issues

### DISCUSSION THREAD

**Discussion by xPolar** [vercel/community#674] [Help] [answered]
Hey there, I have a couple of environment variables I'm using my my serverless functions, however they're all undefined when used, even though when I do `console.log(process.env)` they're shown. How would I fix this?

**Comment by leerob** [comment, marked as answer]
1. Ensure your env vars are in a `.env` file locally when testing
1. Use `vercel dev` to run your application

Then, when deploying to Vercel, ensure the env vars have been added to your project settings. 

https://vercel.com/docs/concepts/projects/environment-variables
